packages/atomic-react/src/components/SearchInterfaceWrapper.tsx

```diff
@@ -18,15 +18,10 @@ interface WrapperProps extends Omit<JSX.AtomicSearchInterface, 'i18n'> {
    */
   onReady?: (executeFirstSearch: ExecuteSearch) => Promise<void>;
   /**
-   * An optional `theme` property can be set in order to load one of the premade Coveo themes.
-   *
-   * Possible values are:
-   *
-   * - `coveo`: the default theme, used if no value is provided. It consists of a set of colors that match the Coveo brand.
-   * - `accessible`: a high contrast theme, best suited for implementations where web accessibility is important.
-   * - `none`: no premade theme will be loaded. You will have to provide the theme yourself.
+   * @deprecated
```

Comment:
That `theme` props was a mistake. It complicates things with many build tool when we try to do dynamic import.

Reply:
We should explain why or offer an alternative after the `deprecated` if possible

Reply:
Yeah, I think the link to the doc that explains how to do theming + the existing doc for Atomic-react should do.